### ISSUE TITLE
Sesh command should deliver embeds in DMs when user logs in

### ISSUE THREAD

**Issue by Kaptard** [ps2gg/ps2.gg#70]
### I'm submitting a... 

- [ ] Bug report
- [x] Feature request
- [ ] Documentation request

### Current behavior 
The sesh command does not track user logins and requires manual user interaction every time they want to play.

### Expected behavior 
Should track the user's characters and send the embeds automatically on login.
This should be enabled by default, with an optional component to disable the notifications.

### Definition of Done 

- [Universal Definition of Done](https://github.com/ps2gg/ps2.gg/blob/master/docs/standards/Definition-Of-Done.md) is adhered to

## 
Depends on #59

**Comment by Kaptard** [comment]
Duplicate of #58
